app/components/Settings/Tabs/Tabs.css

```diff
@@ -6,7 +6,7 @@
     width: 100%;
     /* margin-left: auto; */
     white-space: nowrap;
-    overflow-x: auto;
+    overflow-x: hidden;
```

Comment:
`auto` is needed for when there a bunch of connections open at once. Perhaps you can keep it as `hidden` when there is only one tab and switch it to `auto` when there is more than 1 tab?